packages/lab/stories/tabs.stories.tsx

```diff
@@ -151,11 +151,11 @@ export const Controlled: TabsStory = () => {
     undefined
   );
   const [tabs, setTabs] = useState([
-    "Home",
-    "Transactions",
-    "Loans",
-    "Checks",
-    "Liquidity",
+    { label: "Home", closeable: false },
```

Comment:
Are the changes in this file necessary? They don't look to be restructuring?

Reply:
(+ my other comment) here i just copied the tabs from tabstrip.stories to have the TabDescriptor type